docs/index.md

```diff
@@ -57,6 +57,7 @@ steampipe plugin install googleworkspace
 
 | Item        | Description |
 | :---------- | :-----------|
+| APIs | 1. Go to the [Google API Console](https://console.cloud.google.com/apis/dashboard). <br/> 2. Select the project that contains your credentials. <br/> 3. Click `Enable APIs and Services`. <br/> 4. Enable: `Calendar`, `Drive`, `Gmail`, `People`.
```

Comment:
```suggestion
| APIs | 1. Go to the [Google API Console](https://console.cloud.google.com/apis/dashboard). <br/> 2. Select the project that contains your credentials. <br/> 3. Click `Enable APIs and Services`. <br/> 4. Enable: `Google Calendar API`, `Google Drive API`, `Gmail API`, `Google People API`.
```

When I searched for `Calendar`, `Drive`, etc., in the Enable APIs console, I saw a few results for each, so I think it'd be helpful if we include the exact titles as they appear. Thoughts?

Reply:
Yup, this is better, thanks.